.github/workflows/test.yml

```diff
@@ -47,7 +47,10 @@ jobs:
         tox --verbose
 
     - name: Upload coverage
-      uses: codecov/codecov-action@v3
+      uses: codecov/codecov-action@v4
       with:
-        name: Python ${{ matrix.python-version }}
+        name: coverage-data-${{ matrix.python-version }}
+        path: ".coverage.*"
+        include-hidden-files: true
+        merge-multiple: true
```

Comment:
These aren't parameters of the codecov-action@v4, see here for a list of options: https://github.com/codecov/codecov-action/blob/v4/README.md#arguments

Reply:
@jezdez - thank you for getting this sorted. I do appreciate this is not your day job - perhaps you might accept additional people onto the Jazzband team who have appriop[riate permissions for high level stuff so that there is less of a burden on you as a single point of dependency?

@pauloxnet - can you confirm that the token is stored incsecrets as "CODECOV_TOKEN"

Reply:
@pauloxnet - it seems to be fine... workjing through remaining failures and then hoewpfully we can start to merge the backlog of PR's

Reply:
Thanks @jezdez ...but I still not have access to the settings of this project :-(